fmt.sh

```diff
@@ -107,19 +107,15 @@ echo "\
 # Check for correct copyrights
 run dade-copyright-headers "$dade_copyright_arg" .
 
+# We do test hlint via Bazel rules but we run it separately
+# to get linting failures early.
 if [ "$hlint_diff" = "true" ]; then
     changed_haskell_files="$(git diff --name-only origin/master | grep '.hs$' || [[ $? == 1 ]])"
```

Comment:
I know you didn't do this, but while we're on the subject…

```suggestion
    changed_haskell_files="$(git diff --name-only origin/master | grep '\.hs$' || [[ $? == 1 ]])"
```